org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/contentassist/IdeContentProposalAcceptor.java

```diff
@@ -24,11 +26,25 @@
 public class IdeContentProposalAcceptor
 		implements IIdeContentProposalAcceptor, Comparator<Pair<Integer, ContentAssistEntry>> {
 
+	@Inject
```

Comment:
import missing

Reply:
You are right. Sorry for that. The Eclipse environment I had to move patches from our patched classes to xtext-core is not building properly and  I will need to set it up again.